app/data/jsonSchemaForm/fundingParameterEPUiSchema.ts

```diff
@@ -34,6 +34,7 @@ export const fundingParameterEPUiSchema = {
   holdbackPercentage: {
     "ui:widget": "NumberWidget",
     isPercentage: true,
+    numberOfDecimalPlaces: 2,
```

Comment:
Is it confirmed that it should be 2 places?

Reply:
Added two decimal places per the comment by @Sepehr-Sobhani on the ticket - @nanyangpro can you confirm what the expected number of decimal places are?

Reply:
Update: Confirmed with Nan that 2 decimal places is acceptable.